c/json.c

```diff
@@ -2632,6 +2696,97 @@ static void copyJson(JsonBuilder *builder, Json *parent, char *parentKey, Json *
   }
 }
 
+static void deleteJson(Json *base, const char *deleteKey) {
+  if (deleteKey) {
+    Json *parentNode = NULL;
+    Json *activeNode = base;
+    char* copiedKey = strdup(deleteKey);
+    char workStr[500]; // huge working buffer on the stack for potential keys, all should realistically be <50 chars
```

Comment:
According to [this](https://github.com/zowe/zowe-common-c/blob/5d4f38341119b5e1406fd4ac110c6c74b7472382/c/yaml2json.c#L357), 256 should be fine. The best approach is to use the same `#define` everywhere.

Reply:
Moved the `#define MAX_JSON_KEY` to `json.h` and will use it here.